test/token-sweep.test.ts

```diff
@@ -0,0 +1,197 @@
+// Start - Support direct Mocha run & debug
+import 'hardhat'
+import '@nomiclabs/hardhat-ethers'
+// End - Support direct Mocha run & debug
+
+import chai, {expect} from 'chai'
+import {before} from 'mocha'
+import {solidity} from 'ethereum-waffle'
+import {
+    SweepERC20TokensHarness,
+    SweepERC721TokensHarness,
+    ERC721PresetMinterPauserAutoId,
+    ERC20PresetMinterPauser
+} from '../typechain'
+import {
+    deployContract,
+    deployContractWithProxy,
+    signer
+} from './framework/contracts'
+import {constants, ContractReceipt, Wallet} from 'ethers'
+import {verifyERC20TransferEventLogs} from './contracts/common/verify-erc20-transfer'
+import {successfulTransaction} from './framework/transaction'
+import {verifyERC721TransferEventLogs} from './contracts/common/verify-erc721-transfer'
+
+// Wires up Waffle with Chai
+chai.use(solidity)
+
+const ADDRESS_ZERO = constants.AddressZero
+
+describe('Token Sweep contracts', () => {
+    let beneficary: string
+    let erc20: ERC20PresetMinterPauser
+    const erc20Symbol = 'TKN'
+    const erc20Name = 'Test ERC20 Token'
+    const erc721Symbol = 'TKN721'
+    const erc721Name = 'Test ERC721 Token'
+    const erc721URI = 'blah'
+    let erc20SweepHarness: SweepERC20TokensHarness
+    let erc721: ERC721PresetMinterPauserAutoId
+    let erc721SweepHarness: SweepERC721TokensHarness
+    before(async () => {
+        beneficary = (await signer(2)).address
+
+        erc20SweepHarness =
+            await deployContractWithProxy<SweepERC20TokensHarness>(
+                'SweepERC20TokensHarness'
+                // [erc20Symbol, erc20Name]
+            )
+
+        erc721SweepHarness =
+            await deployContractWithProxy<SweepERC721TokensHarness>(
+                'SweepERC721TokensHarness'
+            )
+
+        erc20 = await deployContract<ERC20PresetMinterPauser>(
+            'ERC20PresetMinterPauser',
+            erc20Symbol,
+            erc20Name
+        )
+
+        erc721 = await deployContract<ERC721PresetMinterPauserAutoId>(
+            'ERC721PresetMinterPauserAutoId',
+            erc721Name,
+            erc721Symbol,
+            erc721URI
+        )
+    })
+    describe('setBeneficiary()', () => {
```

Comment:
nit: avoid method name, instead focus on the purpose to build a natural language sentence when composing the levels of describes and it  e.g. `setBeneficiary()` -> `set beneficiary`